ansys/mapdl/core/jupyter.py

```diff
@@ -88,53 +93,38 @@ def launch_mapdl_on_cluster(
     if "-m " in additional_switches:
         raise ValueError(
             'Memory option "-m" not permitted when launching from the '
-            "kubernetes cluster and is set with the `memory` parameter"
+            "kubernetes cluster and is set with the ``memory`` parameter"
         )
     if "-np " in additional_switches:
         raise ValueError(
             'CPU option "-np" not permitted when launching from the '
-            "kubernetes cluster and is set with the `nproc` parameter"
+            "kubernetes cluster and is set with the ``nproc`` parameter"
         )
 
     # check resources
     nproc = int(nproc)
     if nproc < 0:
-        raise ValueError("Requested CPUs `nproc` must be greater than 0")
+        raise ValueError("Requested CPUs ``nproc`` must be greater than 0")
     if nproc > MAX_CPU:
-        raise ValueError(f"Requested CPUs `nproc` must be less than {MAX_CPU}")
+        raise ValueError(f"Requested CPUs ``nproc`` must be less than {MAX_CPU}")
 
     if memory < 0.25:
-        raise ValueError("Requested memory `mem` must be greater than 0.25")
+        raise ValueError("Requested memory ``mem`` must be greater than 0.25")
     if memory > MAX_MEM:
-        raise ValueError(f"Requested memory `mem` must be less than than {MAX_MEM}")
+        raise ValueError(f"Requested memory ``mem`` must be less than than {MAX_MEM}")
 
-    # convert memory from GB to Mi
+    # # convert memory from GB to Mi
     memory *= 1024
+
     if "-smp" in additional_switches:
-        warnings.warn(
-            'Ignoring additional switch "-smp".  Incompatible with docker ' "container."
+        raise ValueError(
+            'Ignoring additional switch "-smp".  Incompatible with docker container.'
         )
-        additional_switches = additional_switches.replace("-smp", "")
-    additional_switches += f"-m -{memory} -np {nproc}"
 
-    # need a way of making the image user-selectable
-    image = "mapdlhelm.azurecr.io/mapdl:v22.0.0"
-    command = (
-        'printf "" | /ansys_inc/v202/ansys/bin/mapdl %s -smp -grpc -custom /ansys_inc/v202/grpc/ansys.e201t.DEBUG-0.53.1'
-        % additional_switches
-    )
-    env = {"ANSYSLMD_LICENSE_FILE": "1055@10.0.0.16"}
-    ip, pod_name = manager.spawn_pod(
-        image,
-        env=env,
-        cpu=1000 * nproc,
-        memory=memory,
-        command=command,
-        start_timeout=start_timeout,
-        verbose=verbose,
-    )
+    additional_switches += f"-m -{memory} -np {nproc}"
+    args = additional_switches.split()
+    ip, name = manager.spawn_mapdl(version=tag, args=args, verbose=verbose)
```

Comment:
I'm checking:
```suggestion
    ip, name = manager.spawn_mapdl(version=tag, args=args, verbose=verbose)
```
https://github.com/pyansys/ansys-jupyterhub-manager/blob/af80a5711c4882756a9c2c5a264b1efc0e1526f3/ansys/jupyterhub/manager/manager.py#L314

Are you sure the argument `args` is correct?

Reply:
Good work checking this. There's a PR in progress https://github.com/pyansys/ansys-jupyterhub-manager/blob/0efdd9514183b352abce3284ed43aa662bca5f96/ansys/jupyterhub/manager/manager.py#L322-L329 that uses the new `args` argument.